ipamutils/utils_test.go

```diff
@@ -1,16 +1,20 @@
 package ipamutils
 
 import (
-	"testing"
-
 	_ "github.com/docker/libnetwork/testutils"
+	"github.com/stretchr/testify/assert"
+	"testing"
 )
 
 func init() {
-	InitNetworks()
+	prePool1 := &PredefinedPools{"172.80.0.0/16", 24}
+	prePool2 := &PredefinedPools{"172.90.0.0/16", 24}
+	DefaultAddressPools := []*PredefinedPools{prePool1, prePool2}
+	InitNetworks(DefaultAddressPools)
 }
 
 func TestGranularPredefined(t *testing.T) {
+	PredefinedBroadNetworks = nil
```

Comment:
Doesn't this effectively nullify second part of the test?   If that was the intent (it appears this test was misnamed originally), why not just remove the second part of the test?   Also, wouldn't 'nil'ing PredefinedBroadNetworks introduce an ordering dependency between this function and TestInitAddressPools() if both were ever run in the same test run?